entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/model/SolrConcept.java

```diff
@@ -0,0 +1,199 @@
+package eu.europeana.entitymanagement.solr.model;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.solr.client.solrj.beans.Field;
+
+import eu.europeana.entitymanagement.definitions.model.Concept;
+import eu.europeana.entitymanagement.vocabulary.AgentSolrFields;
+import eu.europeana.entitymanagement.vocabulary.ConceptSolrFields;
+import eu.europeana.entitymanagement.vocabulary.EntitySolrFields;
+
+public class SolrConcept extends Concept {
+
+	private String payload;
+	
+	public SolrConcept() {
+		super();
+	}
+
+	public SolrConcept(Concept concept) {
+		super();
```

Comment:
please switch to the use of super(concept) and remove the redundant calls to set the values.

Reply:
Improved.